tests/test_gossip.py

```diff
@@ -130,8 +130,8 @@ def test_announce_address(node_factory, bitcoind):
     l1.daemon.wait_for_log(r"\[OUT\] 0101.*47"
                            "010102030404d2"
                            "017f000001...."
-                           "02000000000000000000000000000000002607"
-                           "04e00533f3e8f2aedaa8969b3d0fa03a96e857bbb28064dca5e147e934244b9ba50230032607")
+                           "02000000000000000000000000000000004d86"
+                           "04e00533f3e8f2aedaa8969b3d0fa03a96e857bbb28064dca5e147e934244b9ba50230034d86")
```

Comment:
Can you explain what these are and why they need to be changed?

Reply:
The commit message says this (and I agree it is very terse)

```
       hex: 2607 is now 4d86
       dec: 9735 is now 19846
```

So in that test, the regtest network is being used and previously even regtest used the default port of `9735` when the port was not set, but now for regtest there is `9735 + 18443 - 8332` (the default port `plus` [rpc_port specific for network](https://github.com/jsarenik/lightning/blob/jsn/default_ports/bitcoin/chainparams.c#L66) `minus` default RPC port (8332, used as a mask here)).

Reply:
@michaelfolkson notice those old lines ending with `2607` and new lines with `4d86`.